README.rst

```diff
@@ -28,13 +28,35 @@ degradation in accuracy. Although all major deep learning frameworks support FP1
 not available today.
 
 TE addresses the problem of FP8 support by providing APIs that integrate with popular Large Language
-Model (LLM) libraries. It provides python layer (initially supporting pyTorch, with support for more
-frameworks in the future) consisting of modules to easily build Transformer layer as well as
-framework agnostic library in C++ including structs and kernels needed for FP8 support. Modules
-provided by TE internally maintain scaling factors and other values needed for FP8 training, greatly
+Model (LLM) libraries. It provides python layer consisting of modules to easily build Transformer
+layer as well as framework agnostic library in C++ including structs and kernels needed for FP8 support.
+Modules provided by TE internally maintain scaling factors and other values needed for FP8 training, greatly
 simplifying for the users.
 
-Transformer Engine in action:
+Support Matrix:
+
++-----------+-----------+
+| Framework | Status    |
++===========+===========+
+|  pyTorch  | Supported |
++-----------+-----------+
+|    JAX    | Supported |
++-----------+-----------+
+
+
+
+Transformer Engine in Action
+----------------------------
+
+General Steps
```

Comment:
I don't like this section - `TE in Action` is not really supposed to accurately explain what happens in the examples, just to show how easy it is to use it. Also we should not give the (false) impression here that TE is just for FP8.

Reply:
Change this section as Examples.